docs/docsite/rst/community/steering/steering_committee_membership.rst

```diff
@@ -42,6 +41,13 @@ A person is eligible to become a Committee member if they have:
 #. Active contributions to  Ansible and/or related projects in any form described in the :ref:`collections_contributions`.
 #. A consent to follow the :ref:`steering_expectations`.
 
+Team membership
+^^^^^^^^^^^^^^^
+
+The Committee can accept a team to be a member.
+In this case, the team chooses its representative and announces the person in a dedicated `Community Topic <https://forum.ansible.com/tags/c/project/7/community-wg>`_.
```

Comment:
Maybe we should explicitly state that the team can rotate persons from time to time? And somehow make clear that announcing the person who's representing the team always in the same thread, not a new dedicated thread every time this happens?

Reply:
+ 1 to mentioning the same thread
For the rotation part: i deliberately didn't put it as imo the important thing here is that they decide themselves and how they do it is their business as long as it's announced in the topic.

Reply:
Added a note about using same thread, PTAL

Reply:
I think we should put in the guidelines 'somewhere' that there is a minimum period any representative must commit to (whether a person in rotation or a regular SC member).  We do have places we need to update so the forum voting works etc.  So perhaps we can add something here or elsewhere in the membership requirements that a member MUST commit to serving for a minimum of three months."  Or something like that.

If y'all feel this is a separate vote, I can start a forum topic on 'minimum serving time' or some such thing.

Reply:
As for using the same thread - that's in the sentence below already...

Reply:
>     * For the rotation part: i deliberately didn't put it as imo the important thing here is that they decide themselves and how they do it is their business as long as it's announced in the topic.

> I think we should put in the guidelines 'somewhere' that there is a minimum period any representative must commit to (whether a person in rotation or a regular SC member). We do have places we need to update so the forum voting works etc. So perhaps we can add something here or elsewhere in the membership requirements that a member MUST commit to serving for a minimum of three months."

Yes, makes (both) sense to me. I think we're OK with people rotating, but they can't do it every couple of days. It would be too much overhead for us to adjust all the permissions. 

> If y'all feel this is a separate vote, I can start a forum topic on 'minimum serving time' or some such thing.

I think we should discuss this now while we're at it, not in a separate thread.

> As for using the same thread - that's in the sentence below already...

Yes, it is now. I think @Andersson007 added this because of my comment ;-)

Maybe we could add there that this shouldn't happen too often. Maybe just add `To avoid too much overhead like changing permissions and similar representative changes shouldn't happen more often than once every three months.` or something like this?

We could also say mustn't instead of shouldn't, but this gives us less flexibility.

Reply:
I added a suggested edit below we can consider. I used two months minimum because we know the current core representative has a hard limit this cycle.

Reply:
We talked about this at the meeting and merged @samccann's suggestion.